web/templates/message_body.hbs

```diff
@@ -5,7 +5,7 @@
         {{> message_avatar ~}}
         <span class="sender_name auto-select" role="button" tabindex="0">
             <span class="sender_name_padding view_user_card_tooltip"></span>
-            <span class="view_user_card_tooltip">{{msg/sender_full_name}}{{> status_emoji msg/status_emoji_info}}</span>
+            <span class="view_user_card_tooltip line-with-status-emoji">{{msg/sender_full_name}}{{> status_emoji msg/status_emoji_info}}</span>
```

Comment:
I assume this is intended to be the `inline-status-emoji` class?

Reply:
Wow, yes that is what it should be. I must've cherry-picked my way back to an earlier, wordier class I started with originally.

Reply:
(This is corrected in the latest push.)